src/patternfly/components/Breadcrumb/breadcrumb.scss

```diff
@@ -4,7 +4,7 @@
   // Breadcrumb Item
   --pf-c-breadcrumb__item--FontSize: var(--pf-global--FontSize--sm);
   --pf-c-breadcrumb__item--FontWeight: var(--pf-global--FontWeight--semi-bold);
-  --pf-c-breadcrumb__item--LineHeight: var(--pf-global--LineHeight--lg);
```

Comment:
Why the change from lg to md?

Reply:
Only because the lg version didn't exist. It's no problem to add it though. Just wasn't sure what the value should be. Also, there's this https://github.com/patternfly/patternfly-next/issues/1109

Reply:
I would go ahead and change this to `var(--pf-global--LineHeight--sm)` since that's what the design specified, and we can change it via #1109 if it should be different.